k8s/pkg/prefixcollector/service.go

```diff
@@ -1,112 +1,109 @@
 package prefixcollector
 
 import (
-	"context"
+	"strings"
 	"sync"
+	"time"
 
-	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
-
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/rest"
 
-	"github.com/networkservicemesh/networkservicemesh/controlplane/api/plugins"
+	"github.com/networkservicemesh/networkservicemesh/sdk/common"
 	"github.com/networkservicemesh/networkservicemesh/sdk/prefix_pool"
 )
 
+const (
+	configMapName = "nsm-config"
```

Comment:
Hardcoded name.

Reply:
This configmap is internal thing, so it's name IMO is not supposed to be set at after-build time, or is it?

Reply:
Hardcoding values of whatever type is not a good practice. This can be the default name, I agree. But the proper practice is to allow it to be changed. Env variable is a good way to do it. Which of course shoudl be well documented.

Reply:
I totally agree with the idea, but there should be a reasonable limit for configure-ability. Many project's container/pod names are hard-coded in *.go files, why make the name of an internal configmap be configurable from environment variable?